psi4/driver/driver_cbs.py

```diff
@@ -1469,44 +2102,38 @@ def cbs(func, label, **kwargs):
 
     # Call schemes for each portion of total energy to 'place orders' for calculations needed
     d_fields = [
-        'd_stage', 'd_scheme', 'd_basis', 'd_wfn', 'd_need', 'd_coef', 'd_energy', 'd_gradient', 'd_hessian', 'd_alpha'
+        'd_stage', 'd_isdelta', 'd_scheme', 'd_basis', 'd_wfn', 'd_component', 'd_need', 'd_coef', 'd_energy',
+        'd_gradient', 'd_hessian', 'd_alpha'
     ]
-    f_fields = ['f_wfn', 'f_basis', 'f_zeta', 'f_energy', 'f_gradient', 'f_hessian', 'f_options']
+    f_fields = ['f_wfn', 'f_component', 'f_basis', 'f_zeta', 'f_energy', 'f_gradient', 'f_hessian', 'f_options']
     GRAND_NEED = []
     MODELCHEM = []
```

Comment:
What is "MODELCHEM", and what is "GRAND_NEED"? Evidently, it's more than a collection of NEED, because it has other fields.

Reply:
Still a ~~magus~~ need, yet much grander. See comment above.